src/thing_traps.c

```diff
@@ -740,8 +740,8 @@ void process_trap_charge(struct Thing* traptng)
             if ((slb->kind != SlbT_CLAIMED) && (slb->kind != SlbT_PATH)) {
                 traptng->health = -1;
             }
-            traptng->rendering_flags &= TRF_Transpar_Flags;
```

Comment:
was this already wrong and now fixed? as clear_flag would be ``&= ~TRF_...`` not the ``&= TRF_...`` here
old code would clear everything that's not transparency flags

Reply:
I did not change the functionality on purpose here.

Reply:
Discussed on discord, we are going to assume this an accidental bugfix. Because I would not know why the original code was as it was.